### ISSUE TITLE
regular and moderator fields not working

### ISSUE THREAD

**Issue by dilushan** [thomasleveil/b3-plugin-customcommands#4]
user, admin, fulladmin. senioradmin, superadmin fields woking fine. But when i put a command in regular and moderator b3 not detecting it. It says unrecognized command. b3 version is 1.9.2

**Comment by danielepantaleone** [comment]
You should post a b3.log file showing the problem :)

**Comment by dilushan** [comment]
160215 23:13:50	DEBUG	"CustomcommandsPlugin: loading command 'aaa'"
160215 23:13:50	VERBOSE	"CustomcommandsPlugin: no help found for command 'aaa'"
160215 23:13:50	DEBUG	'AdminPlugin: Command "aaa (None)" registered with aaa for level (1, 100)'
160215 23:13:50	DEBUG	'CustomcommandsPlugin: loading custom commands for group regular'
160215 23:13:50	DEBUG	"CustomcommandsPlugin: loading command 'aab'"
160215 23:13:50	VERBOSE	"CustomcommandsPlugin: no help found for command 'aab'"
160215 23:13:50	ERROR	'AdminPlugin: unknown group regular'
160215 23:13:50	ERROR	u"AdminPlugin: Cannot register command 'aab'. Bad level/group : 'regular'. Expecting a level (0, 1, 2, 20, 40, 60, 80, 100) or group keyword (guest, user, reg, mod, admin, fulladmin, senioradmin, superadmin)"
160215 23:13:50	DEBUG	'CustomcommandsPlugin: no section [mod commands] found in config file'
160215 23:13:50	DEBUG	'CustomcommandsPlugin: loading custom commands for group admin'
160215 23:13:50	DEBUG	"CustomcommandsPlugin: loading command 'aad'"
160215 23:13:50	VERBOSE	"CustomcommandsPlugin: no help found for command 'aad'"
160215 23:13:50	DEBUG	'AdminPlugin: Command "aad (None)" registered with aad for level (40, 100)'


here is the log related to problem. I managed to fix moderator section by renaming [moderator commands] to [mod commands] in plugin_customcommands.ini. Still commands in regular not working

**Comment by danielepantaleone** [comment]
The keyword for group `regular` is `reg`: http://wiki.bigbrotherbot.net/usage:groups?s[]=groups

**Comment by dilushan** [comment]
```
160215 23:30:30	DEBUG	"CustomcommandsPlugin: loading command 'aaa'"
160215 23:30:30	VERBOSE	"CustomcommandsPlugin: no help found for command 'aaa'"
160215 23:30:30	DEBUG	'AdminPlugin: Command "aaa (None)" registered with aaa for level (1, 100)'
160215 23:30:30	DEBUG	'CustomcommandsPlugin: no section [regular commands] found in config file'
160215 23:30:30	DEBUG	'CustomcommandsPlugin: loading custom commands for group mod'
160215 23:30:30	DEBUG	"CustomcommandsPlugin: loading command 'aac'"
160215 23:30:30	VERBOSE	"CustomcommandsPlugin: no help found for command 'aac'"
160215 23:30:30	DEBUG	'AdminPlugin: Command "aac (None)" registered with aac for level (20, 100)'
160215 23:30:30	DEBUG	'CustomcommandsPlugin: loading custom commands for group admin'
160215 23:30:30	DEBUG	"CustomcommandsPlugin: loading command 'aad'"
160215 23:30:30	VERBOSE	"CustomcommandsPlugin: no help found for command 'aad'"
160215 23:30:30	DEBUG	'AdminPlugin: Command "aad (None)" registered with aad for level (40, 100)'

```

Here is the new log after i changed [regular commands] to [reg commands] and [moderator commands] to [mod commands] in plugin_customcommands.ini. Now aaa, aac, aad working.

**Comment by dilushan** [comment]
I guess there is a code error in regular section. Because neither regular and reg seems to working. Unfortunately i'm not good at python so cant fix it myself. Anyway i will take a look at it

**Comment by dilushan** [comment]
ok fixed it myself. changed in python file

```
for group_name in ("guest", "user", "regular", "mod", "admin", "fulladmin", "senioradmin", "superadmin"):
to 
for group_name in ("guest", "user", "reg", "mod", "admin", "fulladmin", "senioradmin", "superadmin"):

```
now everything fin

**Comment by danielepantaleone** [comment]
Nice :

**Comment by thomasleveil** [comment]
Thank you @dilushan. This is now fixed in version 1.2
